package.json

```diff
@@ -19,18 +20,24 @@
     "@types/react": "^17.0.11",
     "@types/react-dom": "^17.0.7",
     "history": "^5.0.0",
+    "nanoid": "^3.1.23",
     "node-sass": "^6.0.1",
+    "pouchdb": "^7.2.2",
+    "pouchdb-debug": "^7.2.1",
+    "pouchdb-find": "^7.2.2",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-hook-form": "^7.10.1",
     "react-router-dom": "^6.0.0-beta.0",
     "react-scripts": "4.0.3",
     "typescript": "^4.3.2",
-    "web-vitals": "^2.0.1"
+    "web-vitals": "^2.0.1",
+    "yup": "^0.32.9"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
+    "start": "PORT=5500 PUBLIC_URL=ideas react-scripts start",
+    "build": "PUBLIC_URL=ideas react-scripts build",
```

Comment:
You don't need to add it as an environment variable. Just add `"homepage": "/ideas"` in the package.json, and that will automatically set the PUBLIC_URL in react and the `index.html` template.

Reply:
Ok